docs/docs-content/clusters/data-center/nutanix/install-pcg/deploy-kubernetes-cluster-pcg.md

```diff
@@ -96,8 +96,15 @@ following the process described in the
 
 4.  Instantiate Nutanix Cluster API.
 
+    :::info
+
+    To prevent conflicts with the recent Nutanix CAPI provider updates, you need to install this exact version of the
```

Comment:
```suggestion
    To prevent conflicts with the recent Nutanix CAPI provider updates, you need to instantiate Cluster API with this exact version of the
```